is.js

```diff
@@ -102,6 +102,14 @@ var is = function is(x) {
             }
             return currYear - parseInt(x) == THIRTEEN
         },
+        barely: {
+            thirteen: function() {
+                return x == THIRTEEN || (x > THIRTEEN && x < (THIRTEEN + 0.2));
+            }
+        },
+        thirteenthCentury: function() {
+            return x > 1201 && x < 1300;
```

Comment:
Whether or not you believe in the strict or common definition of centuries 1201 should be included in the 13th century.

https://en.wikipedia.org/wiki/Century#Start_and_end_in_the_Gregorian_calendar